pkg/openfeature/util_test.go

```diff
@@ -0,0 +1,21 @@
+package openfeature
+
+// Test Utils
+
+type StateHandlerForTests struct {
```

Comment:
I would not expose this since it is testing code.

Reply:
Fixed with the latest commit